routing/car_router.cpp

```diff
@@ -225,6 +229,7 @@ IRouter::ResultCode FindSingleOsrmRoute(FeatureGraphNode const & source,
   route.SetTurnInstructions(move(turns));
   route.SetSectionTimes(move(times));
   route.SetStreetNames(move(streets));
+  route.SetTraffic({} /* No traffic info in case of OSRM */);
```

Comment:
The route invariant is not violated here. Route may have either empty |m_traffic| or the size of |m_traffic| should be m_poly.GetPolyline().GetSize()-1. This invariant is checked in Route::AppendRoute(Route const & route)/Route::AppendTraffic(Route const & route).

Why do we need to support an empty |m_traffic| vector? Some routes (osrm/pedestrian..) don't have any traffic info at all.